.circleci/config.yml

```diff
@@ -1570,11 +1610,12 @@ workflows:
             packages/contracts-bedrock/scripts/verify
             op-dripper
             devnet-sdk
-            op-acceptance-tests
           requires:
             - contracts-bedrock-build
             - cannon-prestate-quick
           context: circleci-repo-readonly-authenticated-github-token
+      - go-acceptance-tests:
+          context: circleci-repo-readonly-authenticated-github-token
```

Comment:
Just double checking - is this context required?

Reply:
I'm not sure, I used the go-tests as a template.

Reply:
This context is only required when we download mise dependencies, FYI